ddtrace/tracer/option.go

```diff
@@ -801,8 +820,10 @@ func WithProfilerEndpoints(enabled bool) StartOption {
 // WithPartialFlushing enables flushing of partially finished traces.
 // This is done after "numSpans" have finished in a single local trace at
 // which point all finished spans in that trace will be flushed, freeing up
-// any memory they were consuming. This feature is currently disabled by
-// default but may be enabled in the future.
+// any memory they were consuming. This can also be configured by setting
+// DD_TRACE_PARTIAL_FLUSH_ENABLED to true, which will default to 1000 spans
+// unless overriden with DD_TRACE_PARTIAL_FLUSH_MIN_SPANS. Parital flushing
+// is disabled by default.
 func WithPartialFlushing(numSpans int) StartOption {
 	return func(c *config) {
 		c.partialFlushMinSpans = numSpans
```

Comment:
This should probably also set `c.partialFlushEnabled` to true?

Reply:
Yes! Great catch!! I definitely meant to do this. Added it, as well as a test.